networkit/test/test_algebraic.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import unittest
+import os
+import networkit as nk
+import numpy as np
+import scipy
+import random
+
+class TestAlgebraic(unittest.TestCase):
+
+	def testMatrices(self):
+		G = nk.readGraph("input/jazz2_directed.gml",nk.Format.GML)
+		A = nk.algebraic.adjacencyMatrix(G, "dense")
+		self.assertEqual(A[0][0], 1.0)
+		self.assertEqual(A[1][1], 1.0)
+		self.assertEqual(A[2][2], 0.0)
+		self.assertEqual(A[3][3], 0.0)
+		self.assertEqual(A[4][4], 0.0)
+		A2 = nk.algebraic.adjacencyMatrix(G, "sparse")
+		self.assertIsInstance(A2, scipy.sparse.csr_matrix)
+		B = nk.algebraic.laplacianMatrix(G)
+		self.assertIsInstance(B, scipy.sparse.coo_matrix)
+		eigenB = nk.algebraic.laplacianEigenvectors(G)
+		self.assertIsInstance(eigenB[0][0], np.complex128)		
+
+	def testEigenvectorValues(self):
```

Comment:
Here you are just testing that the first eigenvector type is complex. Shouldn't you check that the function produces the expected eigenvectors?